data/hai/unfettered.txt

```diff
@@ -0,0 +1,271 @@
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+# this file contains assets that can be used at any point in the campaign
+
+phrase "Essential Unfettered Mission Fail"
+	word
+		`You have failed an essential Unfettered mission. You may want to reload your previous save if you want to be able to finish the campaign.`
+
+phrase "Essential Unfettered Mission Fail If Killed"
+	word
+		`If you did not kill the dueling ship, you may try this mission again later. Otherwise, you have failed an essential Unfettered mission. You may want to reload your previous save if you want to be able to finish the campaign.`
+
+# using <, > so we can have an empty name
+substitutions
+	"<uhai>" "Triumph of the True Hai"
+	"<, unfettered title>" ", monkey"
+	"<, unfettered title>" ", space-monkey"
+		has "Unfettered Jump Drive 1: offered"
+	"<, unfettered title>" ", Captain"
+		has "unfettered: beaten the captain"
+
+
+phrase "cerberus hails"
+	word
+		"We do not accept your surrender!"
+		"You stand no chance!"
+		"Even if you take me down, my Masters will make quick work of you."
+		"Do not attempt to subdue me."
+		"I obey only the Masters."
+		"They hold my soul! Soon, they will hold yours as well."
+		"Die!"
+		"You have been allowed to live long enough. Now I come to cut the thread of your life."
+		"I may be a pawn, but at least I am on a greater path."
+		"You may never hope to achieve half of what I have done."
+		"My price is far above what you could possibly afford. Do not even attempt to bribe me."
+		"Do you hope to corrupt me? There is no price that is high enough."
+		"I guard the gates of Hell. You will not deter me from that."
+		"You think yourself special? You will die, just like the rest of you ignorant fools!"
+		"There can only be death for the likes of you."
+		"The Masters are superior to you."
+		"You may manage to kill me, but you will never break me."
+		"Run, you fool!"
+		"Killing you will allow me to ascend."
+		"When I will bring the husk of your ship to the Masters, I will be promoted."
+		"I look forward to killing you."
+		"Do you even know what you have gotten yourself into?"
+		"You will never go beyond that simple, mortal, shell!"
+		"Maybe in death you will find peace."
+		# the (reality destroyer/dimensional eraser) will be made safe
+		"Kashalam Eshar!"
+		# you will never be "prime"
+		"Uminor Magresh."
+		# your quest is void (futile)
+		"Porotar Elnam."
```

Comment:
What is the translation cipher?

Reply:
there's none, I made these up

I'm not that hardcore for a few hails haha